src/lib.rs

```diff
@@ -375,11 +395,40 @@ impl fuse::Filesystem for SandboxFS {
         }
         reply.ok();
     }
+
+    fn rename(&mut self, _req: &fuse::Request, _parent: u64, _name: &OsStr, _newparent: u64,
+        _newname: &OsStr, _reply: fuse::ReplyEmpty) {
+        panic!("Required RW operation not yet implemented");
+    }
+
+    fn rmdir(&mut self, _req: &fuse::Request, _parent: u64, _name: &OsStr,
+        _reply: fuse::ReplyEmpty) {
+        panic!("Required RW operation not yet implemented");
+    }
+
+    fn setattr(&mut self, _req: &fuse::Request, _inode: u64, _mode: Option<u32>, _uid: Option<u32>,
```

Comment:
It should, and it will be when I add a `setattr` to the `Node`... but this is just the implementation of a trait function so I have no control over it...